solutions/swb-reference/integration-tests/support/complex/accountHelper.ts

```diff
@@ -16,26 +16,24 @@ export class AccountHelper {
   }
 
   // TODO: Replace with Accounts list API call when it is available
-  public async listOnboardedAccounts(): Promise<{[id: string]: string}[]> {
+  public async listOnboardedAccounts(): Promise<{ [id: string]: string }[]> {
     const queryParams = {
       index: 'getResourceByCreatedAt',
       key: { name: 'resourceType', value: 'account' }
     };
     const response = await this._awsSdk.helpers.ddb.query(queryParams).execute();
-    let accounts: {[id: string]: string}[] = [];
+    let accounts: { [id: string]: string }[] = [];
     if (response && response.Items) {
       accounts = response.Items.map((item) => {
-        return item as unknown as {[id: string]: string};
+        return item as unknown as { [id: string]: string };
       });
     }
     return accounts;
   }
 
   public async deleteDdbRecords(accountId: string, awsAccountId: string): Promise<void> {
-    await this._awsSdk.helpers.ddb
-      .delete({ pk: `ACC#${accountId}`, sk: `ACC#${accountId}` }).execute();
-    await this._awsSdk.helpers.ddb
-      .delete({ pk: `AWSACC#${awsAccountId}`, sk: `ACC#${accountId}` }).execute();
+    await this._awsSdk.helpers.ddb.delete({ pk: `ACC#${accountId}`, sk: `ACC#${accountId}` }).execute();
```

Comment:
Since you're making changes here, could you use constant strings from here: `workbench-core/base/src/constants/resourceTypeToKey.ts`?